staging/src/k8s.io/apiserver/pkg/storage/value/encrypt/envelope/kmsv2/envelope.go

```diff
@@ -242,3 +250,22 @@ func generateKey(length int) (key []byte, err error) {
 
 	return key, nil
 }
+
+// validateAnnotations tests the following:
+//  1. The annotations key is what Kubernetes calls a "qualified name".
+//     This validation is the same as the label key name validation.
+//  2. The size of annotations keys + values is less than 32KB.
```

Comment:
Power of `2` not `10` :smile:

```suggestion
//  2. The size of annotations keys + values is less than 32 kB.
```

Reply:
I blame copilot!!!